console/console-init/ui/src/components/FormAndJson/FormAndJson.tsx

```diff
@@ -41,7 +41,11 @@ const FormAndJson: React.FunctionComponent<IFormAndJsonProps> = ({
   return (
     <>
       {!isValid && <h1 style={{ color: "red" }}>inValid</h1>}
-      <Switch id="iot-enable-switch" isChecked={showJson} onChange={onToggle} />
+      <Switch
+        id="form-json-toggle-switch-button"
```

Comment:
I don't know if it's necessary to have `toggle` keyword there because it's from my POV kind of irrelevant from point of testing, but if that makes sense to you go ahead and let it be with `toggle`

Reply:
You're right. I'll make changes to this id